cleanup/artifactCleanup/artifactCleanup.json

```diff
@@ -10,6 +10,6 @@
     "paceTimeMS": 500,
     "disablePropertiesSupport": true,
     "keepArtifacts": true,
-    "keepArtifactsRegex": "~/.\\_-\\d+\\.\\d+\\.\\d+\\-\\d{2}\\.\\d\\.\\*/"
+    "keepArtifactsRegex": "~/.*[-+]\\d[\\.\\d+]*[-+]\\d[\\.\\d+]*\\.*/"
```

Comment:
maybe stupid question, but what is '+' in [-+] ? do we have artifacts with '+' char in the name? Can you give an example?

Reply:
This is a regex for artifacts using semver.

Example:

`1.2.0-R21.2+4.5.6`

We use that for storing the upstream version of a 3rd party library. In the example above this means: our version of the artifact is `1.2.0-R21.2` which is based on the upstream library `4.5.6`.

Reply:
This regex is meant to keep release artifacts (eg. `foo-1.2.3-R21.3`)